files/es/web/html/element/q/index.html

```diff
@@ -26,24 +26,24 @@
 <table class="properties">
  <tbody>
   <tr>
-   <th scope="row">{{web.link("/es/docs/Web/HTML/Content_categories", "Categorías de contenido")}}</th>
-   <td>{{web.link("/es/docs/Web/HTML/Content_categories#Flow_content", "Flujo de contenido")}}, {{web.link("/es/docs/Web/HTML/Content_categories#Phrasing_content", "redacción de contenido")}}, contenido palpable.</td>
+   <th scope="row"><a href="/es/docs/Web/HTML/Content_categories">Categorías de contenido</a>{{</th>
```

Comment:
It seems that these `{{` are causing the PR test to fail (to the `es` team, I hope it's OK for me to comment here :))

```suggestion
   <th scope="row"><a href="/es/docs/Web/HTML/Content_categories">Categorías de contenido</a></th>
```